.claude/hooks/block_git_no_verify.py

```
@@ -1,58 +1,168 @@
-    sys.exit(0)
+HOOKS_PATH_KEY = "core.hookspath"
 
-block = False
-sanitized = []
-seen_git = False
-seen_commit = False
+# GIT_CONFIG_KEY_0=core.hooksPath のような環境変数経由の指定
+GIT_CONFIG_KEY_RE = re.compile(r"^GIT_CONFIG_KEY_\d+=(.+)$", re.IGNORECASE)
+# 対になる GIT_CONFIG_VALUE_n / GIT_CONFIG_COUNT
+GIT_CONFIG_PAIR_RE = re.compile(r"^GIT_CONFIG_(VALUE_\d+|COUNT)=", re.IGNORECASE)
+# GIT_CONFIG_GLOBAL=/dev/null / GIT_CONFIG_SYSTEM=/dev/null（設定ごと無効化）
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)